bedrock/firefox/templates/firefox/new/batm/machine.html

```diff
@@ -0,0 +1,45 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "firefox/new/batm/base.html" %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_new_batm_machine' %}
+{% endblock %}
+
+{% block body_class %}scene-1 machine-animated{% endblock %}
+
+{% block content %}
+<main role="main">
+  <section class="machine">
+    <video id="machine" preload="auto" autoplay loop muted>
+      <source src="https://assets.mozilla.net/video/batm-animation.webm" type="video/webm">
+      <source src="https://assets.mozilla.net/video/batm-animation.mp4" type="video/mp4">
```

Comment:
It was the hosting. MP4s served directly from bedrock with all our extra security headers were being flagged as unsafe content in Safari. Serving them from this S3 bucket doesn't have all those extra headers so Safari is happy to treat it as safe content... seems counter-intuitive, but oh well.